federation/cluster/federation-up.sh

```diff
@@ -90,7 +90,7 @@ function init() {
       --dns-zone-name="${DNS_ZONE_NAME}" \
       --dns-provider="${DNS_PROVIDER}" \
       --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
-      --apiserver-arg-overrides="--storage-backend=etcd2" \
+      --apiserver-arg-overrides="--storage-backend=etcd2,--runtime-config=api/all=true" \
```

Comment:
@colhom, Thanks for having a look.
I dont think its desirable to include that flag by default in kubefed.
The intent of separating the api types is to ensure they are not enabled by default (when users run the api server).
Kubefed is the current recommended mechanism and IMHO should not "by default" enable those api resources which are not enabled by default.
For test, I guess there would always be a wrapper around it.
And for users who want to use/test these apis, they can always override the default.